modules/seqexec/engine/src/test/scala/seqexec/engine/packageSpec.scala

```diff
@@ -178,15 +177,17 @@ class packageSpec extends FlatSpec with NonImplicitAssertions {
     )
     val p = Stream.eval(IO.pure(Event.start(seqId, user, UUID.randomUUID())))
 
-    executionEngine.process(p)(s0).take(1).compile.last.unsafeRunSync.map(_._2)
+    //take(3): Start, Executing, Paused
+    executionEngine.process(p)(s0).take(3).compile.last.unsafeRunSync.map(_._2)
   }
 
   "sequence state" should "stay as running when action pauses itself" in {
     assert(actionPause.exists(s => Sequence.State.isRunning(s.sequences(seqId))))
   }
 
-  ignore should "change to Paused if output is Paused" in {
-    assert(actionPause.exists(_.sequences(seqId).current.execution.forall(Action.paused)))
+  "engine" should "change action state to Paused if output is Paused" in {
+    val r = actionPause
```

Comment:
This look the same as before. Why was it ignored?

Reply:
It didn't work, because `actionPause` was not running the engine to the point were it would actually process the `Action`.